src/EditorFeatures/CSharpTest/EditAndContinue/Helpers/EditingTestBase.cs

```diff
@@ -78,16 +78,16 @@ public static EditScript<SyntaxNode> GetTopEdits(EditScript<SyntaxNode> methodEd
         /// <summary>
         /// Gets method edits on the current level of the source hierarchy. This means that edits on lower labeled levels of the hierarchy are not expected to be returned.
         /// </summary>
-        internal static EditScript<SyntaxNode> GetMethodEdits(string src1, string src2, MethodKind kind = MethodKind.Regular)
+        internal static EditScript<SyntaxNode> GetMethodEdits(string src1, string src2, MethodKind kind = MethodKind.Regular, string? additionalDeclarations = null)
```

Comment:
Didn't end up using this, but thought I'd leave it in anyway...

Reply:
If it's not used I'd prefer removing it. It's not hard to add back and the code is simpler without it.

---
In reply to: [595690362](https://github.com/dotnet/roslyn/pull/51926#discussion_r595690362) [](ancestors = 595690362)